### PR TITLE
Add JwtHelper class

### PR DESCRIPTION
**Pull request by 2ynn**
This PR adds a `JwtHelper` class that mirrors the [JwtHelper.java](https://github.com/clerk/clerk-sdk-java/blob/main/src/main/java/com/clerk/backend_api/helpers/JwtHelper.java) implementation.

To run the Example usage:
```
poetry install
poetry run helpers/jwthelper.py
```

To run tests:
```
poetry install
poetry run pytest
```

### CODE REVIEW

File: helpers/jwthelper.py [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+            k: v for k, v in {
+                'sub': subject,
+                'aud': audience,
+                'exp': expiration,
+                'nbf': not_before,
+                'iat': issued_at,
+                'azp': azp
+            }.items() if v is not None
+        }
+        return jwt.encode(payload, key, algorithm="HS256")
```
Comment (jescalan) [inline helpers/jwthelper.py:56]: I don't quite understand why we are adding this utility. I don't think there's any scenario where the user should be generating a JWT, clerk does all the generation with our service

**Review by jescalan — COMMENTED** [review]
It seems like this is entirely missing `authenticateRequest`, am I missing something?

File: helpers/jwthelper.py (hunk 1 above)
Comment (2ynn) [reply helpers/jwthelper.py]: I have moved the `generate_token` helper function to the tests and updated the Example usage so you can adapt the payload to your needs

**Comment by 2ynn** [comment]
closing, c.f. #18

**Pull request closed without merging**